spec/models/piece_spec.rb

```diff
@@ -1,59 +1,120 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
-  describe 'obstructed? method' do
-    it "should return true if horizontal right is obstructed" do
-      user = FactoryBot.create(:user)
-      game = FactoryBot.create(:game, user: user)
-      piece = FactoryBot.create(:piece, position_x: 0, position_y: 0, game: game)
-      obstruction = FactoryBot.create(:piece, position_x: 2, position_y: 0, game: game)
-      
-      expect(piece.obstructed?(5, 0)).to eq(true)
-    end
+  
+  let(:user) { FactoryBot.create(:user) }
+  let(:game) { FactoryBot.create(:game, user: user) }
+  
+  describe '#obstructed?' do
+    describe '#horizontal_obstruct' do
     
```

Comment:
ah this is so useful, and i think we should replicate this syntax for piece tests.